addons/stock/wizard/stock_warn_unwanted_replenish.py

```diff
@@ -0,0 +1,23 @@
+from odoo import api, fields, models, Command
+
+class StockWarnUnwantedReplenish(models.TransientModel):
+    _name = 'stock.warn.unwanted.replenish'
+    _description = 'Warn unwanted replenish'
+
+    stock_orderpoint_ids = fields.Many2many('stock.warehouse.orderpoint', string='Stock Orderpoints')
+
+    @api.model
+    def default_get(self, fields_list):
+        defaults_dict = super().default_get(fields_list)
+        stock_orderpoint_ids = self._context.get('active_ids', None)
+        defaults_dict['stock_orderpoint_ids'] = [Command.set(stock_orderpoint_ids)]
+        return defaults_dict
+
+    def action_force(self):
```

Comment:
I would add another action to recompute the qty_to_order and launch the replenishment